cogs/logging/logging.py

```diff
@@ -223,12 +223,18 @@ async def timezone_get(self, ctx, *, user: discord.User = None):
 
         utc_offset = f"UTC{seconds // 3600:+03}:{abs(seconds) // 60 % 60:02}"
 
-        await ctx.send(embed=discord.Embed(
+        result_embed = discord.Embed(
             title="Timezone info",
             description=f"User {user.name}'s timezone is {timezone} ({utc_offset})",
             colour=discord.Colour.blue()
-        ))
-
+        )
+        
+        current_datetime = datetime.now(timezone)
```

Comment:
`datetime` module was imported, use the class.

```suggestion
        current_datetime = datetime.datetime.now(timezone)
```